lib/streamlit/web/server/server.py

```diff
@@ -77,8 +77,16 @@
 # to an unix socket.
 UNIX_SOCKET_PREFIX = "unix://"
 
-# The endpoint we serve media files from.
+# The endpoints
```

Comment:
nit: I think it'd be fine to just remove this comment entirely

Reply:
Removed.